tests/unit/vdom.ts

```diff
@@ -2708,4 +2709,38 @@ describe('vdom', () => {
 			});
 		});
 	});
+
+	describe('behavior', () => {
```

Comment:
This isn’t a great description, also is there any reason this year cannot be in the `create` or `properties` blocks elsewhere

Reply:
Yeah, I didn't know where to put it. Refactoring the test to use `dom.create()` will make it easier to house in that test suite.